package.json

```diff
@@ -175,7 +175,6 @@
   },
   "greenkeeper": {
     "ignore": [
-      "sinon",
```

Comment:
since we're now current and they seem to be better semver now, we can unignore it.